.vscode/dependabot.yml

```diff
@@ -0,0 +1,86 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
I am not familiar with dependabot but seems like daily may be too often. Can we do weekly?

Reply:
Changed to weekly with the latest check-in

Reply:
Why is this in the .vscode folder? I don't think this is related to VS Code.

Reply:
Yes, I agree with Adam, this seems strange

Reply:
Tests have moved to the proper .github folder, per Dependabot instructions: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates